reference.md

```diff
@@ -3,7 +3,5 @@ layout: reference
 ---
 
 ## Glossary
-
-FIXME
-
-{% include links.md %}
+The important parts of a plot are summarized in the image below
+![parts of image](https://matplotlib.org/stable/_images/anatomy.png)
```

Comment:
Minor comment: Ideally the text (here `parts of image`) should describe what the image about, e.g., "Names for different parts of a matplotlib plot" (it is shown if the image link breaks for example)

Reply:
done